docker-compose.yml

```diff
@@ -8,22 +8,22 @@ services:
       args:
         KOMODO_BRANCH: dev
     # image: dragonriders/kmdplatform_komodod_dev:latest
-    image: kmdplatform_komodod_dev
-    restart: on-failure
```

Comment:
Why did you remove this line? We definitely need to autorestart container if process crashed.

Reply:
The way the blaster works, if it fails restarting it will do nothing. It will not blast. There is no use having this here. Its better to have it totally fail, and we can then investigate why. If this is a common occurrence in small scale testing, I will need to look at ways of changing how the blast loops are called. I have a few ideas, but if this very simple way works 90%+ of the time I would rather not change it.